setup.cfg

```
@@ -86,7 +86,7 @@ uvloop =
     uvloop
 
 s3 =
-    s3path
+    s3path==0.3.3
```

I try not a version packages here so we don't lock people.
- Lets at least
```suggestion
    s3path>=0.3.3
```

---

Upps, my bad. I thought I did `>=`. Thanks!